### PR TITLE
fix FIND_IN_SET query for Postgres

### PR DESCRIPTION
**Pull request by StalkAlex**
Also fixing this issue https://github.com/dmstr/yii2-pages-module/issues/16

### CODE REVIEW

**Comment by Quexer69** [comment]
Looks good to me. Sorry @StalkAlex didn't account the good old Postgres.

@schmunk42 can be merged 👍

File: db/traits/ActiveRecordAccessTrait.php [hunk 1]
```diff
@@ -288,4 +290,18 @@ private function addAccessError($action, $attribute)
...
+     * @param $accessRead
+     * @param $authItems
+     * @return string
+     */
+    private static function getInSetQueryPart($accessRead, $authItems)
+    {
+        if (Yii::$app->db->getDriverName() === 'mysql') {
+            return 'FIND_IN_SET(' . $accessRead . ', "' . $authItems . '") > 0';
+        }
+        return " '" . $accessRead . "'= SOME (string_to_array('$authItems', ','))";
```
Comment (schmunk42) [inline db/traits/ActiveRecordAccessTrait.php]: Isn't this PgSQL specific.
What's about a switch statement for the supported DBs, including an exception for non-supported ones?

CC: @StalkAlex @Quexer69

Comment (StalkAlex) [inline db/traits/ActiveRecordAccessTrait.php]: Yep. Those are for mysql and postgres only. I'll add exception and switch.

**Comment by schmunk42** [comment]
Thank you very much!

**Comment by StalkAlex** [comment]
No problem! :)

**Comment by schmunk42** [comment]
@StalkAlex Have a look at https://git.hrzg.de/dmstr/phd5-app/builds/83824 - I've pushed several Postgres fixes in our extensions.

You can try it with Docker by activating `tests/docker-compose.postgres.yml` (master branch)

**Comment by StalkAlex** [comment]
@schmunk42  Ok, I'll give a try in one of the new projects. Now we're using phd4, I've used some docker ideas from phd5, it much more comfortable and simplier. Thanks!

**Comment by StalkAlex** [comment]
Please tag new realease with this fix.

**Comment by schmunk42** [comment]
We've tagged https://github.com/dmstr/yii2-db/releases/tag/0.8.10-rc1

But I think it's also safe to tag https://github.com/dmstr/yii2-db/releases/tag/0.8.10 - done.

**Comment by StalkAlex** [comment]
Great, thanks!

**Pull request merged**